indi-armadillo-platypus/dragonfly_dome.cpp

```diff
@@ -168,8 +168,8 @@ void DragonFlyDome::ISGetProperties(const char *dev)
 {
     INDI::Dome::ISGetProperties(dev);
 
-    defineProperty(&PerPortSP);
-    loadConfig(true, PerPortSP.name);
+    defineProperty(PerPortSP);
+    loadConfig(true, PerPortSP.getName());
```

Comment:
no need to call loadConfig, you can call PerPortSP.load() but the right place to do this is not in ISGetProperties but in after line 148 directly where the property is initialized.

Reply:
sure, using now in line 157